eng/testing/tests.browser.targets

```diff
@@ -166,6 +171,8 @@
               TargetPath="%(WasmFilesToIncludeFromPublishDir.Identity)"
               Condition="'%(WasmFilesToIncludeFromPublishDir.Identity)' != ''" />
     </ItemGroup>
+
+    <WriteLinesToFile File="$(WasmMainHtmlPath)" Lines="&lt;html&gt;&lt;body&gt;&lt;script type='module' src='$(WasmMainJSFileName)'&gt;&lt;/script&gt;&lt;/body&gt;&lt;/html&gt;" Overwrite="True" Condition="!Exists('$(WasmMainHtmlPath)')"/>
```

Comment:
Why do we generate and not copy src/tests/Common/wasm-test-runner/index.html ?

Reply:
Because it can contain a different script than `test-main.js`

Reply:
ahhh this is dynamic